cloudini_lib/tools/src/cloudini_rosbag_converter.cpp

```diff
@@ -36,7 +38,9 @@ int main(int argc, char** argv) {
       ("profile", "Apply a profile to encoding. See '--help' for details. It can be a path to a file or a string",
        cxxopts::value<std::string>())                                 //
       ("c,compress", "Convert PointCloud2 to CompressedPointCloud2")  //
-      ("d,decode", "Convert CompressedPointCloud2 to PointCloud2");
+      ("d,decode", "Convert CompressedPointCloud2 to PointCloud2")    //
+      ("m,method", "Compression method to use when writing data back to mcap ('lz4' or 'zstd', 'none')",
+       cxxopts::value<std::string>()->default_value("none"));
```

Comment:
let's default to zstd
```suggestion
       cxxopts::value<std::string>()->default_value("zstd"));
```